app/routes/admin.js

```diff
@@ -0,0 +1,139 @@
+const Link = require('../../app/services/domain/link')
+const userRoleService = require('../services/user-role-service')
+const UserRole = require('../services/domain/user-role')
+const Authorisation = require('../authorisation')
+const Roles = require('../constants/roles')
+const FieldValidator = require('../services/validators/field-validator')
+const ErrorHandler = require('../services/validators/error-handler')
+
+module.exports = function (router) {
+  router.get('/admin', function (req, res) {
+    var userRole
+    try {
+      if (Authorisation.hasRole(req, Roles.DATA_ADMIN)) {
+        userRole = Roles.DATA_ADMIN
+      } else if (Authorisation.hasRole(req, Roles.SYSTEM_ADMIN)) {
+        userRole = Roles.SYSTEM_ADMIN
+      }
+      return res.render('admin', {
+        title: 'Admin',
+        userRole: userRole
+      })
+    } catch (error) {
+      res.status(401).redirect('/')
```

Comment:
Ideally the admin link would not be visible if you are not an Admin user. This will be part of the permission story. If you are admin depending on your admin type the correct links will be enabled for your level.